docs/install.rst

```diff
@@ -326,6 +320,35 @@ Important to note is that packages on `conda` and `pip` are typically *not* comp
 Frequently asked questions
 --------------------------
 
+**Why are the plots in my notebook not interactive?**
+
+Jupyter notebooks support the use of interactive figures.
+To enable interactive plots, you have to invoke the correct `magic commands <https://ipython.readthedocs.io/en/stable/interactive/magics.html>`_ in the notebook.
+When using Jupyter notebook, the following command will switch the `matplotlib` backend from the inline one (which renders images) to the interactive backend::
+
+    %matplotlib widget
+
+When using JupyterLab, `ipympl` is the only backend that provides interactive plots with Pylake.
```

Comment:
What do you want to tell the user with this comment? That for Jupyter Lab you also need `%matplotlib widget`?

Reply:
I dropped the second sentence. The fact that it uses `ipympl` is probably less interesting to users.